.github/CODEOWNERS

```diff
@@ -1,6 +1,7 @@
 # Marketplace related
 /Tests/Marketplace/approved_tags.json @demisto/content-leaders @michalgold
 /Tests/Marketplace/approved_usecases.json @demisto/content-leaders @michalgold
+/Tests/Marketplace/landingPage_sections.json @demisto/content-leaders @michalgold
```

Comment:
@Bargenish Can we change the naming to stick with the current convention?

Reply:
@ShahafBenYakir  it's a bit different case than the approved tags and usecases so I'm not sure it's right here. but we don't have any limitations to change it.

Reply:
I meant the file name, it is landingPage_sections and should be landing_page_sections